File: reference source/xml2scanner/XML2Scanner/_catalog.vb
Public Class _catalog
    'Package Count
    Private m_pkgcnt As Integer

    Public Property Cpkgcnt() As Integer

        Get

            Return m_pkgcnt

        End Get

        Set(ByVal value As Integer)

            m_pkgcnt = value

        End Set

    End Property

    ' Catalog CRC

    Private m_pkgcrc As String

    Public Property Cpkgcrc() As Integer

        Get

            Return m_pkgcrc

        End Get

        Set(ByVal value As Integer)

            m_pkgcrc = value

        End Set

    End Property

    'Package Name
    Private m_pkgname As Integer

    Public Property Cpkgname() As String

        Get

            Return m_pkgname

        End Get

        Set(ByVal value As String)

            m_pkgname = value

        End Set

    End Property
End Class

File: reference source/xml2scanner/XML2Scanner/_catalog.cs
using Microsoft.VisualBasic.CompilerServices;

namespace XML2Scanner
{
    public class _catalog
    {
        // Package Count
        private int m_pkgcnt;

        public int Cpkgcnt
        {

            get
            {

                return m_pkgcnt;

            }

            set
            {

                m_pkgcnt = value;

            }

        }

        // Catalog CRC

        private string m_pkgcrc;

        public int Cpkgcrc
        {

            get
            {

                return Conversions.ToInteger(m_pkgcrc);

            }

            set
            {

                m_pkgcrc = value.ToString();

            }

        }

        // Package Name
        private int m_pkgname;

        public string Cpkgname
        {

            get
            {

                return m_pkgname.ToString();

            }

            set
            {

                m_pkgname = Conversions.ToInteger(value);

            }

        }
    }
}